### DISCUSSION TITLE
make the elements unclickable when the they hide?

### DISCUSSION THREAD

**Discussion by hooke007** [tomasklaen/uosc#134] [General]
E.g. In idle mode, even though users cannot see the volume bar, we could still change it by touching the "hiding elements".

Due to limitations of the `no-border` , I always use border. I used to be confused that clicking on the right cornor of the output would close the mpv, then realised that the area was for the hiding uosc "buttons".

P.s. There is one bug I found, when `window-maximized=yes` and `border=yes` , uosc would render its own border.

**Comment by tomasklaen** [comment]
These are valid issues, why discussion?

Regardless, both issues should be fixed now on master.

**Reply by hooke007** [reply]
It seems to be existed for a long time, I'm not sure if it was treated as "bugs".
